ui/raidboss/timeline.js

```diff
@@ -51,6 +51,7 @@ class Timeline {
       if (!r[replaceKey])
         continue;
       let keys = Object.keys(r[replaceKey]);
+      keys.sort().reverse();
```

Comment:
I thought about this a bit over the week, and I think this is not ideal and it would be better to do this ordering inside the translation block.

Javascript key ordering is predictable, so I think it's better to have the translation section be an ordered set of replacements.  Sorting here removes people's control.

If you have replacements that should be reordered, then I think we should just reorder them in the timelineReplace section, rather than trying to figure out the best ordering later.  So, I think this sorting, and both bits of sorting in check_translation.py should be removed.

Reply:
If JavaScript dictionary has predictable keys order, just order them in the Jason section is a good choice I think.
I thought it acts like python, which is unpredictable lol

Reply:
If the keys are all strings that cannot be represented as integers (e.g. not "1") then Object.keys will returns you keys in insertion order: https://www.stefanjudis.com/today-i-learned/property-order-is-predictable-in-javascript-objects-since-es2015/.  I am pretty sure the version of Chrome that we are using also behaves this way for Object.keys, so you can depend on that.

Maybe it means that the Python code for check_translation.py needs to be a little bit different to make sure these stay in order, but for the timeline stuff at runtime, that order can be depended on.